regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -104,20 +105,23 @@ var ChildView = Backbone.View.extend({
                 if (_.isEmpty(this.activeSection) || (this.activeSection !== this.$sections[i].id)) {
                     this.activeSection = this.$sections[i][0].id;
                     this.$activeSection = $(this.$sections[i][0]);
-                    // **Event** trigger active section change
-                    HeaderEvents.trigger('section:open', this.activeSection);
-                    MainEvents.trigger('paragraph:active', this.activeSection);
-
-                    if (typeof window.history !== 'undefined' && typeof window.history.replaceState !== 'undefined') {
-                        // update hash in url
-                        window.history.replaceState(
-                            null,
-                            null,
-                            window.location.origin + window.location.pathname + window.location.search + '#' + this.activeSection
-                        );
+                    if (this.$activeSection.is(':visible')) {
```

Comment:
Okay -- should I make an issue? Silently failing like this seems like a mistake.